server/src/main/java/org/opensearch/action/support/nodes/BaseNodesRequest.java

```diff
@@ -119,6 +127,14 @@ public void setConcreteNodes(DiscoveryNode[] concreteNodes) {
         this.concreteNodes = concreteNodes;
     }
 
+    public void sendDiscoveryNodes(boolean value) {
```

Comment:
nit: could be better named as `setSendDiscoveryNodes`. and getter as `getSendDiscoveryNodes`.

also should we use the name `includeDiscoveryNodes` as the flag instead of `sendDiscoveryNodes`?

Reply:
Sure Arpit, can update the getter and setter, did not change it earlier as I thought it would make reading easier. Will update the variable name as well.

Thanks!